src/current_thread.rs

```diff
@@ -1,25 +1,25 @@
 //! Execute tasks on the current thread
 //!
-//! [`CurrentThread`] provides an executor that keeps futures on the same thread
+//! This module implements an executor that keeps futures on the same thread
 //! that they are submitted on. This allows it to execute futures that are
-//! `!Send`. For more details on general executor concepts, like executing
+//! not `Send`. For more details on general executor concepts, like executing
 //! futures, see [here].
 //!
-//! Before being able to execute futures onto [`CurrentThread`], an executor
-//! context must be setup. This is done by calling [`run`].  From within that
-//! context, [`CurrentThread::execute`] may be called with the future to run in
-//! the background.
+//! Before being able to spawn futures with this module, an executor
+//! context must be setup. This is done by calling either [`block_on_all`] or
```

Comment:
There is no block_on_all fn in the module. This fn is referenced a number of other spots as well.